spring-kafka/src/main/java/org/springframework/kafka/retrytopic/RetryTopicConfigurer.java

```diff
@@ -221,14 +221,18 @@
 
 	private final BeanFactory beanFactory;
 
+	private final RetryTopicNamesProviderFactory retryTopicNamesProviderFactory;
+
 	public RetryTopicConfigurer(DestinationTopicProcessor destinationTopicProcessor,
-						ListenerContainerFactoryResolver containerFactoryResolver,
-						ListenerContainerFactoryConfigurer listenerContainerFactoryConfigurer,
-						BeanFactory beanFactory) {
+								ListenerContainerFactoryResolver containerFactoryResolver,
+								ListenerContainerFactoryConfigurer listenerContainerFactoryConfigurer,
+								BeanFactory beanFactory,
+								RetryTopicNamesProviderFactory retryTopicNamesProviderFactory) {
 		this.destinationTopicProcessor = destinationTopicProcessor;
```

Comment:
Our convention for multi-line signatures is to add a blank line before the code.